backend/scripts/import_doc_postfix/import_doc_postfix.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import sys
+from app.db.models.document import Document
+from app.db.session import SessionLocal
+
+from app.core.validation.cclw.law_policy.process_csv import (
+    validated_input,
+    import_id_from_csv_row,
+    POSTFIX_FIELD,
+)
+
+if __name__ == "__main__":
+    print("Importing document postfixes...")
+    db = SessionLocal()
+
+    if len(sys.argv) != 2:
+        print("Require a CSV file to import")
+        sys.exit(1)
+
+    csv_reader = validated_input(open(sys.argv[1]))
+    mappings = [
+        {
+            "import_id": import_id_from_csv_row(row),
+            "id": db.query(Document.id)
+            .filter(Document.import_id == import_id_from_csv_row(row))
```

Comment:
I tried to do this in one up-front query but reducing the array got messy - and this way works :smile: 

Happy to hear any suggestions you might have on improving this though :+1:

Reply:
Hmmmm, I imagined we'd get all the document objects in one query, sort the list by the id part of the import id, then run through the list looking for duplicates...

Reply:
Sorry about my last comment, I thought I was looking at the relationships script... I should try to sleep! I'm still a little confused about how this works though... why are we building an ID to postfix mapping, when we can create an `import_id` to `postfix` mapping directly from the CSV, and we have a database index on `import_id`. We really never need to touch database level ids at all do we?

Reply:
Yeah we could do it that way... I just liked how you could reduce it down to one single "bulk" update call, if you reference your updated by PK.

Reply:
so we go...
- csv -> import_id
- import_id -> id

The last step so we can make one call to update everything with the postfix info